sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -0,0 +1,454 @@
+    "    enable_online_store=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To confirm that your FeatureGroup has been created we use `DescribeFeatureGroup` and `ListFeatureGroups` APIs to display the created FeatureGroup."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_feature_group.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session.boto_session.client('sagemaker', region_name=region).list_feature_groups() # We use the boto client to list FeatureGroups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Ingest data into a feature group\n",
+    "\n",
+    "After the FeatureGroups have been created, we can put data into the FeatureGroups by using the `PutRecord` API. It will take < 1min to ingest data both of these FeatureGroups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def check_feature_group_status(feature_group):\n",
+    "    status = feature_group.describe().get(\"FeatureGroupStatus\")\n",
+    "    while status == \"Creating\":\n",
+    "        print(\"Waiting for Feature Group to be Created\")\n",
+    "        time.sleep(5)\n",
+    "        status = feature_group.describe().get(\"FeatureGroupStatus\")\n",
+    "    print(f\"FeatureGroup {feature_group.name} successfully created.\")\n",
+    "    \n",
+    "check_feature_group_status(customers_feature_group)\n",
+    "check_feature_group_status(orders_feature_group)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.ingest(\n",
+    "    data_frame=customer_data, max_workers=3, wait=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_feature_group.ingest(\n",
+    "    data_frame=orders_data, max_workers=3, wait=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using an arbirary customer record id, 573291 we use `get_record` to check that the data has been ingested into the feature group."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_id = 573291\n",
+    "sample_record = sagemaker_session.boto_session.client('sagemaker-featurestore-runtime', region_name=region).get_record(FeatureGroupName=customers_feature_group_name, RecordIdentifierValueAsString=str(customer_id))"
```

Comment:
Is it better to move the boto client initialization to the beginning of the notebook? Same as the one used to `list_feature_groups()`.

Reply:
Good question. I was debating the placement of the initialization but after some discussion with others I think that since we only use two calls from `boto`, both would require it's own initialization e.g., for`'sagemaker-featurestore-runtime'` and `'sagemaker'`  incurring a few extra lines of code. But I think more importantly, this way we give the customer a clear understanding between `boto` calls and ones that are from the Python SDK and what the differences are. Often there is some confusion between the two. Let me know your thoughts.